workflows/templates/inspection-results-template.yaml

```diff
@@ -57,9 +57,7 @@ spec:
           cnt=0
           IFS=$'\n' ; for name in $resource_names; do
             let "cnt+=1";
-            kubectl logs $name |\
-              tee "${results}/${cnt}-output" |\
-              jq -r '.stdout."@result"?' > "${results}/${cnt}-result"
+            kubectl logs $name > "${results}/${cnt}-output"
```

Comment:
We should `tee` the output, so that it is visible in the pod log.

Reply:
We see it in the inspection job pod. IMHO it would be better to report what is gathered in the results gathering task as that is the purpose of that task.

Reply:
OK, fair enough, it's kind of a debugging feature but as this is just a script, it's not hard to run the same thing locally if debugging is necessary.

Reply:
The type of debugging you've just described can gather logs from all pods running in the cluster within the flow. Each task should report only what's going on in that task. No need to produce thousands of lines just to see results of parent tasks and hide what's going on in the task ;) (: